.github/workflows/CD.yml

```diff
@@ -103,3 +127,102 @@ jobs:
           aws s3 cp $rpm_artifact s3://artifacts.opendistroforelasticsearch.amazon.com/downloads/rpms/opendistro-knnlib/
           aws s3 cp $deb_artifact s3://artifacts.opendistroforelasticsearch.amazon.com/downloads/debs/opendistro-knnlib/
           aws cloudfront create-invalidation --distribution-id E1VG5HMIWI4SA2 --paths "/*"
+
+  library-build-and-ship-artifacts-arm64:
+    needs: [Provision-Runners]
+    runs-on: [self-hosted, Linux, ARM64, odfe-knnlib-build-arm64]
+    name: Build and release JNI library artifacts arm64
+    strategy:
+      matrix:
+        java: [14]
+        compiler: [g++]
+    steps:
+      - name: Checkout k-NN
+        uses: actions/checkout@v1
+        with:
+          submodules: true
+
+      - name: Configure AWS Credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          aws-region: us-east-1
+
+      # Arm64 is not supported by setup-java actions as of now
+      # Placeholder in case it adds support later
+      # Using pre-installed jdk on arm64 self-hosted runners now
```

Comment:
What version of Java will this be?

Reply:
As of now the runner is CentOS8 with JDK15.